src/general/IDifficulty.cs

```diff
@@ -44,6 +44,11 @@ public interface IDifficulty : IRegistryAssignable
     /// </summary>
     public bool PassiveReproduction { get; }
 
+    /// <summary>
+    ///   Whether the patch map should be automatically fully revealed
+    /// </summary>
+    public bool PatchMapExploration { get; }
```

Comment:
Could you clarify with the documentation which of the following is true:
- To me PatchMapExploration being true would mean that you need to explore the map
- The documentation says the opposite i.e. if this is true then the map is revealed

Changing this property to be called something like `MapIsAlwaysFullyRevealed` would remove the potential confusion.